doc/content/reference/email-templates/available.md

```diff
@@ -17,14 +17,14 @@ Email validation | `validate` | Sent when a user is added as a collaborator of a
 
 The following fields can be used inside all of the email templates:
 
-Field | Description
-------|------------
-`User.ID` | The identifier of the user.
-`User.Name` | The full name of the user.
-`User.Email` | The email of the user.
-`Network.Name` | The full name of the network running on the stack.
-`Network.IdentityServerURL` | The base URL of the Identity Server.
-`Network.ConsoleURL` | The base URL of the Console.
-`Entity.Type` | The type of the entity, such as `application`, `client`, `gateway`, `organization` or `user`.
-`Enitity.ID` | The identifier of the entity.
-`Contact.Type` | The contact type of the recipient, such as `technical`, `billing` or `abuse`.
+Field | Description | Notes
+------|-------------|------
+`User.ID` | The identifier of the user. | If the recipient is the contact of the target entity, the ID will be empty.
+`User.Name` | The full name of the user. | If the recipient is the contact of the target entity, the Name will be `user`.
```

Comment:
This is not very clear to me.

Should we have a matrix per field and template identifier which fields are available?